### PR TITLE
docs: rewrite getting-started for ISO

### LINKED ISSUE #3016: Document ISO install
We need a clear and concise tutorial on installing Talos with the ISO.

### PR DESCRIPTION
**Pull request by Ulexus**
Update the Getting Started documentation to reflect the new ISO-based
installation method.

Fixes #3016

Signed-off-by: Seán C McCord <ulexus@gmail.com>

# Pull Request

## Acceptance

Please use the following checklist:

- [ ] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [ ] you ran conformance (`make conformance`)
- [ ] you formatted your code (`make fmt`)
- [ ] you linted your code (`make lint`)
- [ ] you generated documentation (`make docs`)
- [ ] you ran unit-tests (`make unit-tests`)

> See `make help` for a description of the available targets.

### CODE REVIEW

File: website/content/docs/v0.9/Introduction/getting-started.md [hunk 1]
```diff
@@ -3,55 +3,325 @@ title: Getting Started
...
+For self-built media and network booting, you can use the kernel and initramfs:
+
+- X86: [https://github.com/talos-systems/talos/releases/download/v0.9.0-beta.0/boot-amd64.tar.gz](https://github.com/talos-systems/talos/releases/download/v0.9.0-beta.0/boot-amd64.tar.gz)
+- ARM64: [https://github.com/talos-systems/talos/releases/download/v0.9.0-beta.0/boot-ard64.tar.gz](https://github.com/talos-systems/talos/releases/download/v0.9.0-beta.0/boot-ard64.tar.gz)
+
+When booted from the ISO, Talos will run in RAM, and it will not install itself
+until it is provided a configuration.
+Thus, it is safe to boot the ISO onto any machine.
+
+### Alternative booting
```
Comment (smira) [inline website/content/docs/v0.9/Introduction/getting-started.md:53]: nit: `B`ooting (header capitalization)

**Review by smira — DISMISSED** [review]
I think this the type of documentation we need - complete walkthrough

File: website/content/docs/v0.9/Introduction/getting-started.md [hunk 2]
```diff
@@ -65,21 +335,47 @@ Endpoints _do_, however, need to be members of the same Talos cluster as the
...
 
-To verify what node(s) you're currently talking to, you can run:
+Some people also like to set a default set of nodes in the `talosconfig`.
+This can be done in the same manner, replacing `endpoint` with `node`.
+If you do this, however, know that you could easily reboot the wrong machine
+by forgetting to declare the right one explicitly.
+Worse, if you set several nodes as defaults, you could, with one `talosctl upgrade`
+command upgrade your whole cluster all at the same time.
+It's a powerful tool, and with that comes great responsibility.
+The author of this document does not set a default node.
```
Comment (andrewrynhard) [inline website/content/docs/v0.9/Introduction/getting-started.md:370]: 😄

**Review by andrewrynhard — APPROVED** [review]

File: website/content/docs/v0.9/Introduction/getting-started.md (hunk 2 above)
Comment (andrewrynhard) [reply website/content/docs/v0.9/Introduction/getting-started.md:370]: > I think this the type of documentation we need - complete walkthrough

Agree. This is awesome.

**Review by smira — DISMISSED** [review]

**Review by smira — APPROVED** [review]

**Pull request merged**